src/game.cpp

```diff
@@ -1681,6 +1681,67 @@ ReturnValue Game::internalTeleport(Thing* thing, const Position& newPos, bool pu
 }
 
 //Implementation of player invoked events
+void Game::playerEquipItem(uint32_t playerId, uint16_t spriteId)
+{
+	Player* player = getPlayerByID(playerId);
+	if (!player) {
+		return;
+	}
+
+	Item* item = nullptr;
+	if (!(item = player->getInventoryItem(CONST_SLOT_BACKPACK))) {
+		return;
+	}
+
+	Container* backpack = nullptr;
+	if (!(backpack = item->getContainer())) {
+		return;
+	}
```

Comment:
This start is all out of consistency, it does not have a pattern.

Throughout the code, usually you will see
```
Player* player = getPlayerById(playerId);
if (player == nullptr) {
    return;
}
```
So I ask you do it accordingly.